src/tools/r2rdump/UnwindInfo.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace R2RDump
+{
+    struct UnwindCode
+    {
+        public byte CodeOffset { get; }
```

Comment:
@nattress / @jkotas Since the data type in native is a union, would it be preferred to have the UnwindCode struct have an explicit field layout to make this field access similar?

Reply:
C# does not have bit-level explicit field layout that you would need here. I think reading it as byte is fine.